src/app/pages/join-session/join-session.component.html

```diff
@@ -10,7 +10,14 @@
         </mat-form-field>
         <mat-form-field appearance="outline" class="form-field">
           <mat-label>Name</mat-label>
-          <input matInput class="name"  formControlName="nameControl">
+          <input matInput class="name" formControlName="nameControl">
+        </mat-form-field>
+        <mat-form-field appearance="outline" class="form-field">
+          <mat-label>Nest Barcode</mat-label>
+          <input matInput inputmode="numeric" pattern="[0-9]*" class="nest" formControlName="nestControl">
+          <mat-error *ngIf="sessionFormGroup.controls['nestControl'].invalid">
```

Comment:
I think it would be a good idea to add a hint here, something like this:
```suggestion
          <input matInput inputmode="numeric" pattern="[0-9]*" class="nest" formControlName="nestControl">
          <mat-hint>The number at the top of your nest marker</mat-hint>
          <mat-error *ngIf="sessionFormGroup.controls['nestControl'].invalid">
```